.github/workflows/test.yml

```diff
@@ -82,13 +82,19 @@ jobs:
             !*.md
 
       - name: Go related changes
-        ## Warning: When we add Integration tests to this workflow
-        # we need to add shim/ directory to this as well
         id: go
         uses: technote-space/get-diff-action@v6
         with:
           PATTERNS: |
             cli/**
+      - name: Go E2E related changes
+        id: go_e2e
+        uses: technote-space/get-diff-action@v6
+        with:
+          PATTERNS: |
+            shim/**
```

Comment:
shim is no longer needed. We should probably add `turbo-updater` too

Reply:
removing shim, but can we rename turbo-updater to turborepo-updater instead? Why the naming inconsistency?